tests/integration/test_oci.py

```diff
@@ -0,0 +1,128 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
```

Comment:
you can remove test_stub.py now :-)

Reply:
this PR is very much a work-in-progress, don't blame me if you look at the code and turn into stone!